src/intent.html

```diff
@@ -295,16 +295,17 @@ <h3 id="mixing_intent_properties">Intent Properties</h3>
    <h3 id="intent_using">Using Intent Concepts and Properties</h3>
    <p>When the <code class="attribute">intent</code> attribute corresponding to a specific node
      contains a concept component, possibly with properties and arguments,
-     the AT's [=Intent Concept Dictionary=] should be consulted.
+     the supported [=Intent Concept Dictionary=] entries should be consulted.
      The concept name should be normalized
      (<q>`_`</q> (U+00F5) and <q>`.`</q>  (U+002E) to <q>`-`</q> (U+002D)),
      and compared using <a data-cite="INFRA#ascii-case-insensitive">ASCII case-insensitive</a>
      match.</p>
-   <p>An concept is considered a <dfn id="intent_known_concept">known concept</dfn> (to the AT)
-     when the normalized name,
-     the fixity property (which may be defaulted in the concept dictionary),
+   <p>A concept is considered a <dfn id="intent_known_concept">known concept</dfn> (to the AT)
+     when the AT has any internal mapping recognizing that concept.
+     It is also considered a <dfn id="intent_matched_concept"></dfn>matched concept</dfn> when
+     its normalized name, the fixity property (which may be defaulted in the concept dictionary),
      and the arity (number of arguments, if any)
-     all match an entry in the AT's concept dictionary.
+     all match an entry in the Core or Open concept dictionaries.
```

Comment:
No sorry I think this a complete change in the meaning of the core and open lists. There has never been any expectation that a system reads the open list, it is just guidance for implementors on names to use.  Thus the classification of a concept as known/unknown (or matched/unmatched) which affects its fallback rendering as a literal, can not depend on the open list, it has to, as in the current draft, and Bruce's PR, depend on the system-specific names that are matched by the AT in use.

Reply:
Excuse me, but I have personally had a system that read the Open list back in 2021. What exactly are you basing your claims on?

The current spec text says [here](https://w3c.github.io/mathml/#intent-open):
>  AT reading MathML attributed with a name in this list MAY use the speech hints provided by the intent definition

That is an expectation that AT _may_ read the open list. My changes here are consistent with the current text.

Reply:
the fact that there is no expectation that systems do something does not mean that no system does.

Your system importing the full list at just means that that system's implemented dictionary includes all of the open list, which is of course allowed.  But your text says that the interpretation of a concept name depends on whether it "match an entry in the Core or Open concept dictionaries" and that means that adding an entry to the open list affects whether a system is treated as known or unknown, and that can't be the case. A concept is just known (or matched) if a system has code to handle it.

Reply:
> the fact that there is no expectation that systems do something does not mean that no system does.

Exactly, completely agree. I think you have misread my text honestly. It has happened to me too when I rush to reply 9 minutes after something gets posted online.

> But your text says that the interpretation of a concept name depends on whether it "match an entry in the Core or Open concept dictionaries"

Actually no, this is what you've misread. My PR text continues to talk about behavior anchored at "known" and "unknown", which I have made _more relaxed_ as they no longer carry any implication of "a dictionary". 

The new "matched"/"unmatched" categories don't (yet?) have any further behavior implications besides becoming terms that anchor the matching against the concept lists.
This is part of what can be further group discussion.

And thank you for your last comment as it includes the perfect case study of the mental model I am attempting to course correct:
> that system's implemented dictionary

That should not be a category of note, to anyone but an implementer that has implemented "a dictionary" in their system. It constrains the AT space too strongly and prevents experimentation. That is exactly why I am motivated to present this PR to the group.

Reply:
again the dictionary does not constrain an implementation in any way, it is a defined term in the specification.

Reply:
> the dictionary does not constrain an implementation in any way, it is a defined term in the specification.

Great. And this PR clarifies the text to pursue that design goal further.

Reply:
If you think that is not clear then we can certainly discuss where to add a clarification  (I don't see why it isn't clear , but don't object to clarification) but that is not what this PR does, it completely redefines the categorisation of concept names from being based on the system-implemented code to being based on the published open and core lists. that is explicitly the one line diff highlighted at the top of this comment thread.

That is a major and unwelcome change.  We should be able to accept contributions to the open list at any time and with essentially no cost, they are suggestions for concept names and speech hints that implementors may choose to implement. Adding a name to that list should not affect the categorisation of concepts in documents.

Reply:
> it completely redefines the categorisation of concept names from being based on the system-implemented code

This has been the definition of "known concept" before this PR, and continues to be the definition with the text in this PR. The text you've highlighted pertains to a new distinction I am drawing for _a kind of_ known concept, called "a matched concept", but known concepts are as before.

Reply:
That is not a distinction that we should make as there are no testable differences that can be made about concept names that are or are not in the open list as there is no requirement for any software to read that list.   Introducing two almost identical things "known" and "matched" would anyway be confusing and should be avoided but since the new matched definition has no meaning at all unless the system reads othe open and core lists and has no specified effects anyway then it is not clear at all why you want to introduce them.

why do you say "matched" is a _kind of known concept_  ? True you have defined it under "known" but if it did prove useful to have a term for "any concept in the open or core lists" that would most naturally include both known and unknown concept names for most systems. But currently theer is no use for such a categorisation why introduce the term at all?

Moreover you have not only introduced a definition of this unused "matched" classification you hav echanged the definition of "known" so it no longer iincludes arity/

Reply:
> Moreover you have not only introduced a definition of this unused "matched" classification you have changed the definition of "known" so it no longer iincludes arity.

As part of my work in the WG, a "known" concept has never depended on arity. If that decision was ever made, it is not reflected in the current text as I read it, and I am unaware of the vote that brought it in.

Current "known concept" spec text is:
> A known concept matches a name in an [Intent Concept Dictionary](https://w3c.github.io/mathml/#dfn-intent-concept-dictionary) recognized by the AT.

Reply:
yes and matching (in the part you deleted) depends on arity. That's just the natural definition, a system may implemet sonething for " 3 factorial" but if given `factorial{a,b,c,d)` with four arguments it's perfectly natural to say it doesn't handle that.

Reply:
Ah OK, so we have had a long-standing disagreement hidden in the way the text was written.

All the better to address it when discussing here, thanks for pointing it out!

Reply:
not really hidden, discussed at length over many meetings and the description of the matching algorithm clarified in the spec as a group decision. You have a long standing objection to using arity which has never been made clear other than you have an objection.

Reply:
Ah yes, I have an objection because I have an objection. That's a good one!

The reason to want the MathML Intent spec to be implementation-agnostic is to avoid pigeonholing the spec to only reflect the implementation choices of early adopters. I've had a prototype of an AT which treated "known concepts" without requiring a predefined arity, and have explained additional implementation avenues which will not comply with a tightly worded definition around arity.

We should focus on the Intent syntax, and seeding enough concepts for getting implementers started quickly, _without_ requiring "known concepts" to follow a predefined data model. Concepts come from the vernacular of natural scientific discourse, and can be "known" to an AT without additional constraints.

Reply:
> Ah yes, I have an objection because I have an objection. That's a good one!
> 
> The reason to want the MathML Intent spec to be implementation-agnostic is to avoid pigeonholing the spec to only reflect the implementation choices of early adopters.

As Bruce has also noted the suggested changes here are forcing more implementation details because (even if that was not your intention) the wording implies that the open list is read.

> I've had a prototype of an AT which treated "known concepts" without requiring a predefined arity, and have explained additional implementation avenues which will not comply with a tightly worded definition around arity.


that is of course allowed: it just means that your implementation matches any arity.  You could also have an implementation that matches every name. It is completely up to the system which names it supports. the current wording places no constraints on this at all.

> 
> We should focus on the Intent syntax, and seeding enough concepts for getting implementers started quickly, _without_ requiring "known concepts" to follow a predefined data model. Concepts come from the vernacular of natural scientific discourse, and can be "known" to an AT without additional constraints.

Exactly so.